lib/cocoapods-fix-react-native/versions/0_55_4-post.rb

```diff
@@ -95,7 +95,7 @@ def detect_missing_subspecs
   # Move Fishhook to be based on RN's imports
   websocket = 'Libraries/WebSocket/RCTReconnectingWebSocket.m'
   websocket_old_code = 'import <fishhook/fishhook.h>'
-  websocket_new_code = 'import <React/fishhook.h>'
+  websocket_new_code = 'import "fishhook.h"'
```

Comment:
This change should not be needed; `<React/fishhook.h>` should work and actually it works for me.

Reply:
Hmm, can I ask what your Podfile looks like? I'm having compilation issues where XCode can't find `<React/fishhook.h>`.

Reply:
Just like the one described in https://facebook.github.io/react-native/docs/integration-with-existing-apps.html

```
# The target name is most likely the name of your project.
target 'NumberTileGame' do

  # Your 'node_modules' directory is probably in the root of your project,
  # but if not, adjust the `:path` accordingly
  pod 'React', :path => '../node_modules/react-native', :subspecs => [
    'Core',
    'CxxBridge', # Include this for RN >= 0.47
    'DevSupport', # Include this to enable In-App Devmenu if RN >= 0.43
    'RCTText',
    'RCTNetwork',
    'RCTWebSocket', # Needed for debugging
    'RCTAnimation', # Needed for FlatList and animations running on native UI thread
    # Add any other subspecs you want to use in your project
  ]
  # Explicitly include Yoga if you are using RN >= 0.42.0
  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'

  # Third party deps podspec link
  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'

end
```

Reply:
Maybe frameworks vs libs?

Reply:
I haven't had issues with either, framework or lib.